guide/12-enrich-data-with-thematic-information/part6_country_enrich.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
```

Comment:
<p>We are going to use the same variables for enrichment that we used in Part 1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>